filewatch/filters.go

```diff
@@ -500,6 +500,31 @@ func (f *FilterManager) launchFollowers(fpath string, deleteState bool) (ok bool
 	return
 }
 
+func (f *FilterManager) checkState(wf watchedFile) (hasWork bool, err error) {
+	//get base dir
+	fname := filepath.Base(wf.pth)
+	fdir := filepath.Dir(wf.pth)
+	var si *int64
+	//swing through all filters and launch a follower for each one that matches
```

Comment:
comment says "launch a follower" which is inaccurate. Would appreciate a comment at the top of the function, too.